rest/snippets/Epg/BroadcastSlotEntity.yml

```diff
@@ -43,4 +43,63 @@ BroadcastSlotEntity:
           type: integer
           example: 1000
           description: Duration of the broadcast slot in seconds.
-      # TODO: Add metadata relationships
+        commercial_break_duration_seconds:
```

Comment:
please check the payload definition for this. This seems to be crooked in the api doc page.
<img width="533" alt="Screenshot 2020-03-09 at 11 12 04" src="https://user-images.githubusercontent.com/2223527/76198701-35768e80-61f7-11ea-9365-22f0f0bb0993.png">

Reply:
There are 2 BroadcastSlotEntities (Epg and Schedule), they are now sharing the same BroadcastSlotResource. The only difference is the `Epg/BroadcastSlotEntity.yml` does not contain relationship, example. https://confluence.aminocom.com/display/MAPI/Recordings+v4#/Recordings/getMostPopularBroadcastSlots. It is intended to add the attributes to the getMostPopularBroadcastSlots response.